### PR TITLE
#184 Unit test 

### PR DESCRIPTION
**Pull request by aayusharora**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+    // Continuous Integration mode
+    // if true, Karma captures browsers, runs the tests and exits
+    singleRun: false,
+
+    // Concurrency level
+    // how many browser should be started simultaneous
+    concurrency: Infinity
+  })
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l84)

File: karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l85)

File: karma.conf.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        './src/config.js',
+        './src/appComponents/**/*',
+        './src/test/unit/spec1.js',
+         './src/appComponents/common/TwitterJsonFactory.js',
+        './src/test/unit/tweetspec1.js'
+     ],
+
+
+    // list of files to exclude
+    exclude: [
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Unquoted property `exclude` found.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l39)

File: karma.conf.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+
+    // list of files / patterns to load in the browser
+    files: [
+        './bower_components/angular/angular.js',
+        './bower_components/angular-mocks/angular-mocks.js',
+        './bower_components/leaflet/leaflet.*',
+        './bower_components/**/angular-simple-logger.js',
+        './bower_components/**/angular-animate.js',
+        './bower_components/**/angular-aria.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l23)

File: karma.conf.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
+// Karma configuration
+// Generated on Fri May 06 2016 17:58:51 GMT+0530 (IST)
+
+module.exports = function(config) {
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing space before function parentheses.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: karma.conf.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+        './bower_components/**/angular-animate.js',
+        './bower_components/**/angular-aria.js',
+        './bower_components/**/angular-leaflet-directive.js',
+        './bower_components/**/angular-material.js',
+        './bower_components/**/angular-material.css',
+        './bower_components/**/angular-ui-router.js',
+        './bower_components/**/ngStorage.js',
+        './src/app.js',
+        './src/config.js',
+        './src/appComponents/**/*',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l31)

File: karma.conf.js [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+    },
+
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['progress'],
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l52)

File: karma.conf.js [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+        './bower_components/**/angular-leaflet-directive.js',
+        './bower_components/**/angular-material.js',
+        './bower_components/**/angular-material.css',
+        './bower_components/**/angular-ui-router.js',
+        './bower_components/**/ngStorage.js',
+        './src/app.js',
+        './src/config.js',
+        './src/appComponents/**/*',
+        './src/test/unit/spec1.js',
+         './src/appComponents/common/TwitterJsonFactory.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l33)

File: karma.conf.js [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+        './bower_components/angular/angular.js',
+        './bower_components/angular-mocks/angular-mocks.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l19)

File: karma.conf.js [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+        './bower_components/leaflet/leaflet.*',
+        './bower_components/**/angular-simple-logger.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2531572274/issues/source?bid=3233857&fileBranchId=3284028#l21)

**Comment by aayusharora** [comment]
@juslee ,@championswimmer , travis.yml is not configured with karma . Should it be done something like this http://stackoverflow.com/questions/19255976/how-to-make-travis-execute-angular-tests-on-chrome-please-set-env-variable-chr

File: karma.conf.js (hunk 1 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l84)

File: karma.conf.js (hunk 2 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l85)

File: karma.conf.js (hunk 3 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Unquoted property `exclude` found.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l39)

File: karma.conf.js (hunk 4 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l23)

File: karma.conf.js (hunk 5 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing space before function parentheses.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: karma.conf.js (hunk 6 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l31)

File: karma.conf.js (hunk 7 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l52)

File: karma.conf.js (hunk 8 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l33)

File: karma.conf.js (hunk 9 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l19)

File: karma.conf.js (hunk 10 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l21)

File: src/appComponents/twitterwall/singletweet.js [hunk 11]
```diff
@@ -55,9 +55,9 @@ function tweet($timeout,$rootScope) {
     }
     function changeTweet() {
 
-      if (scope.tweetNumber !=== 90) {
+      if (scope.tweetNumber !== 90) {
             scope.tweetNumber = scope.tweetNumber + 1;
-            scope.selectedid  =scope.array[scope.tweetNumber].id_str;
+            scope.selectedid  = scope.array[scope.tweetNumber].id_str;
```
Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Multiple spaces found before '='.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189597/issues/source?bid=3233857&fileBranchId=3284028#l60)

Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Identifier 'id_str' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189597/issues/source?bid=3233857&fileBranchId=3284028#l60)

Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Expected indentation of 10 space characters but found 12.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189597/issues/source?bid=3233857&fileBranchId=3284028#l60)

File: karma.conf.js [hunk 12]
```diff
@@ -0,0 +1,87 @@
+// Karma configuration
+// Generated on Fri May 06 2016 17:58:51 GMT+0530 (IST)
+
+module.exports = function (config) {
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189585/issues/source?bid=3233857&fileBranchId=3284028#l4)

Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing function expression name.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189585/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: karma.conf.js [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+    // if true, Karma captures browsers, runs the tests and exits
+    "singleRun": false,
+
+    // Concurrency level
+    // how many browser should be started simultaneous
+    "concurrency": Infinity
+
+  });
+
+}
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189585/issues/source?bid=3233857&fileBranchId=3284028#l87)

File: karma.conf.js [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+      "./bower_components/**/angular-aria.js",
+      "./bower_components/**/angular-leaflet-directive.js",
+      "./bower_components/**/angular-material.js",
+      "./bower_components/**/angular-material.css",
+      "./bower_components/**/angular-ui-router.js",
+      "./bower_components/**/ngStorage.js",
+      "./src/app.js",
+      "./src/config.js",
+      "./src/appComponents/**/*",
+      "./src/test/unit/spec1.js",
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Unexpected trailing comma.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189585/issues/source?bid=3233857&fileBranchId=3284028#l33)

File: karma.conf.js (hunk 13 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189585/issues/source?bid=3233857&fileBranchId=3284028#l87)

File: src/test/unit/spec1.js [hunk 15]
```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
+  var $controller;
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Unexpected var, use let or const instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189603/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: src/test/unit/spec1.js [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
+
+    $rootScope = $injector.get('$rootScope');
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533189603/issues/source?bid=3233857&fileBranchId=3284028#l9)

File: karma.conf.js (hunk 13 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300322/issues/source?bid=3233857&fileBranchId=3284028#l87)

File: karma.conf.js (hunk 12 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300322/issues/source?bid=3233857&fileBranchId=3284028#l4)

Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing function expression name.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300322/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: karma.conf.js (hunk 13 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300322/issues/source?bid=3233857&fileBranchId=3284028#l87)

File: karma.conf.js (hunk 14 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Unexpected trailing comma.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300322/issues/source?bid=3233857&fileBranchId=3284028#l33)

File: src/test/unit/spec1.js [hunk 17]
```diff
@@ -0,0 +1,26 @@
...
+
+  var $rootScope;
+      $controller;
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
+
+    $rootScope = $injector.get('$rootScope');
+    $controller = $injector.get('$controller');
+    $scope = $rootScope.$new();
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Variable '$scope' implicitly declared.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300340/issues/source?bid=3233857&fileBranchId=3284028#l11)

File: src/test/unit/spec1.js [hunk 18]
```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
+      $controller;
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
+
+    $rootScope = $injector.get('$rootScope');
+    $controller = $injector.get('$controller');
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Variable '$controller' implicitly declared.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300340/issues/source?bid=3233857&fileBranchId=3284028#l10)

File: src/test/unit/spec1.js [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+    $scope = $rootScope.$new();
+
+  }));
+  beforeEach(inject(function ($controller) {
+    AppController = $controller("AppController");
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Variable 'AppController' implicitly declared.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300340/issues/source?bid=3233857&fileBranchId=3284028#l15)

File: src/test/unit/spec1.js [hunk 20]
```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
+      $controller;
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
+
+    $rootScope = $injector.get('$rootScope');
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300340/issues/source?bid=3233857&fileBranchId=3284028#l9)

File: src/test/unit/spec1.js [hunk 21]
```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
+      $controller;
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300340/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: karma.conf.js (hunk 12 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664911/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: karma.conf.js (hunk 13 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664911/issues/source?bid=3233857&fileBranchId=3284028#l87)

File: karma.conf.js (hunk 12 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing function expression name.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664911/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: karma.conf.js (hunk 14 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Unexpected trailing comma.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664911/issues/source?bid=3233857&fileBranchId=3284028#l33)

File: karma.conf.js (hunk 13 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664911/issues/source?bid=3233857&fileBranchId=3284028#l87)

File: src/test/unit/spec1.js (hunk 15 above)
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Unexpected var, use let or const instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: src/test/unit/spec1.js [hunk 22]
```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Unexpected var, use let or const instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l3)

File: src/test/unit/spec1.js (hunk 15 above)
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Variable '$controller' should be initialized on declaration.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: src/test/unit/spec1.js [hunk 23]
```diff
@@ -0,0 +1,26 @@
...
+  }));
+  beforeEach(inject(function ($controller) {
+    AppController = $controller("AppController");
+
+  }));
+
+    it("It will describe the title variable", function () {
+
+    expect(AppController.appTitle).toBe(config.title);
+    expect(openevent.totalDays).toBe(0);
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: ["expect" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l22)

File: src/test/unit/spec1.js [hunk 24]
```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l1)

File: src/appComponents/twitterwall/singletweet.js [hunk 25]
```diff
@@ -57,7 +57,7 @@ function tweet($timeout,$rootScope) {
 
       if (scope.tweetNumber !== 90) {
             scope.tweetNumber = scope.tweetNumber + 1;
-            scope.selectedid  =scope.array[scope.tweetNumber].id_str;
+            scope.selectedid  = scope.array[scope.tweetNumber].id_str;
             $rootScope.$broadcast("selectedid",scope.selectedid);
             callchangetweet();
```
Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Expected indentation of 8 space characters but found 12.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879975/issues/source?bid=3233857&fileBranchId=3284028#l62)

File: karma.conf.js (hunk 12 above)
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Missing function expression name.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879963/issues/source?bid=3233857&fileBranchId=3284028#l4)

Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879963/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: src/test/unit/spec1.js [hunk 26]
```diff
@@ -0,0 +1,31 @@
+
+
+
+
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [More than 2 blank lines not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l4)

File: src/test/unit/spec1.js [hunk 27]
```diff
@@ -0,0 +1,31 @@
...
+    $controller = $injector.get("$controller");
+    $scope = $rootScope.$new();
+
+  }));
+  beforeEach(inject(function ($controller) {
+    AppController = $controller("AppController");
+
+  }));
+
+    it("It will describe the title variable", function () {
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Missing function expression name.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l23)

File: src/test/unit/spec1.js [hunk 28]
```diff
@@ -0,0 +1,31 @@
+
+
+
+
+describe("app", function () {
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Missing function expression name.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l5)

File: src/test/unit/spec1.js [hunk 29]
```diff
@@ -0,0 +1,31 @@
...
+
+  let $rootScope;
+  let $controller;
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: ["beforeEach" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l11)

File: src/test/unit/spec1.js [hunk 30]
```diff
@@ -0,0 +1,31 @@
...
+
+    $rootScope = $injector.get("$rootScope");
+    $controller = $injector.get("$controller");
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Identifier '$controller' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l14)

File: src/test/unit/spec1.js [hunk 31]
```diff
@@ -0,0 +1,31 @@
...
+
+describe("app", function () {
+
+  let $rootScope;
+  let $controller;
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
+
+    $rootScope = $injector.get("$rootScope");
```
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Identifier '$rootScope' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l13)

File: src/test/unit/spec1.js (hunk 27 above)
Comment (juslee) [inline src/test/unit/spec1.js]: Codacy Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l23)

File: karma.conf.js (hunk 12 above)
Comment (aayusharora) [inline karma.conf.js]: @juslee , please help me in these kind of issues. They don't seem to be relevant .

File: src/test/unit/spec1.js (hunk 27 above)
Comment (aayusharora) [inline src/test/unit/spec1.js]: @juslee , Can we do something to allow anonymous functions ?

**Pull request merged**